skopeo.spec.rpkg

```diff
@@ -95,6 +95,10 @@ export CGO_CFLAGS+=" -m64 -mtune=generic -fcf-protection=full"
 
 LDFLAGS=""
 
+%if 0%{?rhel}
+rm -rf vendor/github.com/containers/storage/drivers/register/register_btrfs.go
+%endif
+
 export BUILDTAGS="$(hack/libdm_tag.sh)"
 %if 0%{?rhel}
 export BUILDTAGS="$BUILDTAGS exclude_graphdriver_btrfs btrfs_noversion"
```

Comment:
So, rhel9 builds work with the current BUILDTAGS setup, but not rhel8. I suspect it could be the golang version because internal rhel8 builds using golang v1.19.2 are successful without removing that file, while the copr builds are still on golang v1.18.4. I can look into this further

Reply:
The `exclude_graphdriver_btrfs` build is supposed to disable the `register_btrfs.go` file, on versions of Golang going as far back as I remember.

I’m surprised if that fails, and I’d feel more comfortable if the cause were understood (otherwise the ahack/libdm_tag.sh` call and the other tags here are in question).

Reply:
made some changes which seem to work for all envs and have listed couple TODOs [perhaps for 2023]. PTAL.